steward/src/allocation.rs

```diff
@@ -148,6 +154,11 @@ pub fn to_allocation(
     pair_id: String,
     eth_gas_price: u64,
 ) -> somm::Allocation {
+    let salt = &generate_salt();
+    let salt = match std::str::from_utf8(salt) {
```

Comment:
This implementation can possibly generate any valid UTF8 string, which is going to contain lots of characters that could be non-printable, hard to differentiate, etc.

Here is an example of taking a limited subset of characters that I proposed in an earlier PR:

https://github.com/PeggyJV/steward/pull/35#discussion_r770039091

Reply:
Fixed with this commit https://github.com/PeggyJV/steward/pull/57/commits/75849001e98d44b5c4309c707b52489196652085